OpenRA.Mods.Common/Traits/BotModules/BotModuleLogic/BaseBuilderQueueManager.cs

```diff
@@ -36,6 +36,8 @@ class BaseBuilderQueueManager
 		int cachedBuildings;
 		int minimumExcessPower;
 
+		bool productOnce = false;
```

Comment:
Think this could be named better. Maybe something like "itemQueuedThisTick"?

Reply:
Fixed, good idea.